mappings/net/minecraft/world/loot/LootManager.mapping

```diff
@@ -7,5 +7,19 @@ CLASS cog net/minecraft/world/loot/LootManager
 	FIELD f supplierNames Ljava/util/Set;
 	METHOD a getSupplierNames ()Ljava/util/Set;
 	METHOD a toJson (Lcoe;)Lcom/google/gson/JsonElement;
+		ARG 0 supplier
+	METHOD a (Lcof;Lqs;Lcoe;)V
```

Comment:
Hmm, you should probably map method's names when you map the parameters

Reply:
It's a lambda, so you can't name it, but you can map the parameters.